docs/Commands.md

```diff
@@ -378,6 +378,12 @@ Searching for books with "Python" in any TEXT attribute, returning the price sto
 FT.SEARCH books-idx "python" RETURN 3 $.book.price AS price
 ```
 
+Searching for books with textual similarity to some book using VECTOR attribute, returning the top 10 results, sorted by distance.
```

Comment:
Searching for books with semantically similar "title" to "Planet Earth". 
Return top 10 results sorted by distance

Assume, there is a "title_embedding" VECTOR field and you have a VECTOR representation of the query string "Planet Earth" as a BLOB.